cloud/workspace/workspace.go

```
@@ -314,6 +315,18 @@ func Delete(id string, out io.Writer, client astrocore.CoreClient) error {
 }
 
 func selectWorkspace(workspaces []astrocore.Workspace) (astrocore.Workspace, error) {
+	if len(workspaces) == 0 {
+		return astrocore.Workspace{}, nil
```

nit: might have to handle this case on the caller side where err is nil, but the workspace is also empty. Or else we can simply return an error from here

---

Yes, i am just adding that and a test case